antispam_bee.php

```diff
@@ -1222,7 +1222,9 @@ public static function replace_comment_field_callback( $matches ) {
 		$id_script = '';
 		if ( ! empty( $matches['id1'] ) || ! empty( $matches['id2'] ) ) {
 			$output   .= 'id="' . self::get_secret_id_for_post( self::$_current_post_id ) . '" ';
-			$id_script = '<script data-noptimize type="text/javascript">document.getElementById("comment").setAttribute( "id", "a' . substr( esc_js( md5( time() ) ), 0, 31 ) . '" );document.getElementById("' . esc_js( self::get_secret_id_for_post( self::$_current_post_id ) ) . '").setAttribute( "id", "comment" );</script>';
+			if (! self::_is_amp()) {
```

Comment:
There are some space missing:
`if (! self::_is_amp()) {` needs to be `if ( ! self::_is_amp() ) {`, I think.

Reply:
:+1: fixed.